lib/blockchain/install.js

```diff
@@ -97,13 +97,17 @@ function plugin (crypto) {
 }
 
 function run () {
+  const ETH_CRYPTO_CODE ='ETH'
```

Comment:
Since ``ETH`` is in the constant name itself, I don't think this extra redirection is worth it. Could just use the string ``ETH`` when needed.

Reply:
Since this constant is used two more times on this specific function, I think it is a good practice to define it once, and use the defined variable wherever I need. Otherwise would need to change it in several places if such change would be required.
On the other hand, I don't think that ethereum will change its acronym from ETH to something else...

Reply:
good point.